internal/cmd/iam/command.go

```diff
@@ -29,6 +29,9 @@ func New(cfg *v1.Config, prerunner pcmd.PreRunner) *cobra.Command {
 		cmd.AddCommand(newPoolCommand(prerunner))
 		cmd.AddCommand(newProviderCommand(prerunner))
 	}
+	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.group-mapping", dc.Context(), v1.CliLaunchDarklyClient, true, false) {
```

Comment:
Have you made this feature flag yet, or do you need me to make it?
```suggestion
	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.iam.group-mapping", dc.Context(), v1.CliLaunchDarklyClient, true, false) {
```

Reply:
I need you to make it haha